The code/Task2.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+
+class Task {
+  String descriptask;
+  bool isCompleted;
+
+  Task(this.descriptask, this.isCompleted);
```

Comment:
use standard constructor instead
```suggestion
  Task({
        required this.descriptask, 
        required this.isCompleted
        });
```